console/src/components/ApiExplorer/ApiRequest.js

```diff
@@ -1,6 +1,14 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
+import jwt from 'jsonwebtoken';
+
+import Modal from 'react-bootstrap/lib/Modal';
```

Comment:
Let us implement a common modal at the root at the app that subscribes to title, content and `isOpen` in redux. Then we can change these properties from anywhere and the modal becomes reusable.

Reply:
Hey I am not able to wrap my head around this pattern where `CommonModal` can be used as 

`this.props.dispatch({ type: COMMON_MODAL_OPEN, title: <react-element>, content: <react-element>})`

instead of 
```
<CommonModal title={ title }>
    <Body>
</CommonModal>
```

The body can be as complicated as it can. @wawhal Let me know what you think.